src/js/components/agency/visualizations/obligated/ObligatedGraph.jsx

```diff
@@ -22,14 +24,46 @@ export default class AgencyObligatedGraph extends React.Component {
         const obligatedValue = this.props.obligatedAmount;
         const authorityValue = this.props.budgetAuthority;
         const remainderValue = authorityValue - obligatedValue;
+        const outlayValue = this.props.outlay;
 
         let obligatedPercent = 0;
+        let outlayPercent = 0;
+        let remainderPercent = 0;
 
-        if (obligatedValue !== 0 && authorityValue !== 0) {
-            obligatedPercent = (obligatedValue / authorityValue);
+        if ((obligatedValue > 0) && (outlayValue > 0) && (authorityValue > 0)) {
```

Comment:
This seems like a pretty confusing way to approach this problem, given the large number of conditionals.

It looks like what we are essentially doing for authority, obligated, and outlay is dividing their value by the max value of the three or returning 0 if the value is negative. Why not just do that? `Math.max` to get the max value, divide each by the max, then use `Math.min` to ensure that negative values are rounded up to 0. The largest value, being divided by itself, will still come out to 1.

Reply:
Simplified this logic using `Math.max`